src/web/client/test/integration/errorHandler.test.ts

```diff
@@ -0,0 +1,636 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
check the file correctly I still see naming convention not followed

Reply:
Sorry it was a old PR so Will take care of it